src/main/java/com/trailfinder/service/HikeServiceStub.java

```diff
@@ -55,13 +55,7 @@ public void saveProfile(ProfileDTO profile) {
 		{
 			ProfileDTO profile = new ProfileDTO(i, "test " + i, "TestBio " + i, null);
 			
-			EventDTO event = new EventDTO();
-			event.setEventId(i);
-			event.setLatitude(39.1330 + i);
-			event.setLongitude(-84.4134 + i);
-			event.setDistance(i + (i/2));
-			event.setEventCreator(profile);
-			events.add(event);
+			events.add(new EventDTO(i, 39.1330 + i, null, null, -84.4134 + i, i + (i/2), profile, null));
```

Comment:
Used overloaded constructor to initialize new EventDTOs and add them to events.

Reply:
I'd leave this as it were, for documentation purposes.  Reason being, the setters are more descriptive, where you have to make assumptions on what "null, null, i + (i/2)" means.